packages/openai/src/transcription/openai-transcription-options.ts

```diff
@@ -24,6 +24,16 @@ export const openAITranscriptionProviderOptions = z.object({
    */
   prompt: z.string().optional(),
 
+  /**
+   * The format of the output, in one of these options: json, text, srt, verbose_json, or vtt.
+   * For gpt-4o-transcribe and gpt-4o-mini-transcribe, the only supported format is json.
+   * @default 'json'
+   */
+  responseFormat: z
+    .enum(['json', 'text', 'srt', 'verbose_json', 'vtt'])
+    .default('json')
+    .optional(),
```

Comment:
most formats would break our provider?

another option would be to always use `verbose_json` and not make this configurable